newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -343,8 +343,8 @@ export const GameDetailsDialog = ({
           >
             <Tab label={<Trans>Details</Trans>} value="details" />
             <Tab label={<Trans>Builds</Trans>} value="builds" />
-            <Tab label={<Trans>Feedbacks</Trans>} value="feedbacks" />
-            <Tab label={<Trans>Analytics</Trans>} value="analytics" />
+            <Tab label={<Trans>Player Feedback</Trans>} value="feedback" />
+            <Tab label={<Trans>Statistics</Trans>} value="analytics" />
```

Comment:
"Analytics" are what businesses and enterprises use.

"Statistics" are what games use (sports, video games).  This sounds much more "fun".  :)